core/src/main/java/com/sequenceiq/cloudbreak/converter/OrganizationRequestToOrganizationConverter.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.converter;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.model.OrganizationRequest;
+import com.sequenceiq.cloudbreak.domain.security.Organization;
+import com.sequenceiq.cloudbreak.domain.security.User;
+import com.sequenceiq.cloudbreak.service.AuthenticatedUserService;
+import com.sequenceiq.cloudbreak.service.user.UserService;
+
+@Component
+public class OrganizationRequestToOrganizationConverter extends AbstractConversionServiceAwareConverter<OrganizationRequest, Organization> {
+
+    @Inject
+    private AuthenticatedUserService authenticatedUserService;
+
+    @Inject
+    private UserService userService;
+
+    @Override
+    public Organization convert(OrganizationRequest json) {
+        User user = userService.getOrCreate(authenticatedUserService.getCbUser());
+
+        Organization organization = new Organization();
+        organization.setName(json.getName());
```

Comment:
Possible NPE? I didn't find the validation for ```OrganizationRequest```

Reply:
It cannot be null, because " @Size(max = 100, min = 5, ..."